File: src/test/java/no/nav/syfo/testhelper/UserConstants.java
package no.nav.syfo.testhelper;

import no.nav.syfo.mock.AktorMock;

import static no.nav.syfo.mocks.BrukerprofilMock.PERSON_ETTERNAVN;
import static no.nav.syfo.mocks.BrukerprofilMock.PERSON_FORNAVN;

public class UserConstants {

    public static final String ARBEIDSTAKER_FNR = "12345678912";
    public static final String ARBEIDSTAKER_AKTORID = AktorMock.Companion.mockAktorId(ARBEIDSTAKER_FNR);
    public static final String LEDER_FNR = "12987654321";
    public static final String LEDER_AKTORID = AktorMock.Companion.mockAktorId(LEDER_FNR);
    public static final String VIRKSOMHETSNUMMER = "1234";
    public static final String NAV_ENHET = "0330";
    public static final String VEILEDER_ID = "Z999999";

    public final static String PERSON_NAVN = PERSON_FORNAVN + " " + PERSON_ETTERNAVN;
}

File: src/test/kotlin/no/nav/syfo/testhelper/UserConstants.kt
package no.nav.syfo.testhelper

object UserConstants {
    const val ARBEIDSTAKER_FNR = "12345678912"
    val ARBEIDSTAKER_AKTORID = mockAktorId(ARBEIDSTAKER_FNR)
    const val LEDER_FNR = "12987654321"
    val LEDER_AKTORID = mockAktorId(LEDER_FNR)
    const val VIRKSOMHETSNUMMER = "123456789"
    const val NAV_ENHET = "0330"
    const val VEILEDER_ID = "Z999999"
    const val PERSON_TLF = "12345678"
    const val PERSON_EMAIL = "test@nav.no"

    const val PERSON_NAME_FIRST = "First"
    const val PERSON_NAME_MIDDLE = "Middle"
    const val PERSON_NAME_LAST = "Last"
    const val PERSON_NAVN_FULL = PERSON_NAME_FIRST + PERSON_NAME_MIDDLE + PERSON_NAME_LAST

    const val STS_TOKEN = "123456789"
}

private const val MOCK_AKTORID_PREFIX = "10"

fun mockAktorId(fnr: String): String {
    return "$MOCK_AKTORID_PREFIX$fnr"
}
